commit 24ae786e78d211330a7e62d0decc208888fe023f (refs/changes/82/1482/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-09T11:32:42-08:00 (2 years, 8 months ago)
    
    TOOLS-1688 Begone, foul cdspell, lord of DWIM! Leave the typos in peace!

diff --git a/etc/root.bashrc b/etc/root.bashrc
index ce2e8d3..bb269a8 100644
--- a/etc/root.bashrc
+++ b/etc/root.bashrc
@@ -1,5 +1,5 @@
 if [ "$PS1" ]; then
-  shopt -s checkwinsize cdspell extglob histappend
+  shopt -s checkwinsize extglob histappend
   alias ll='ls -lF'
   HISTCONTROL=ignoreboth
   HISTIGNORE="[bf]g:exit:quit"
